unitTests/rest/BackOffRequestTest.ts

```diff
@@ -0,0 +1,65 @@
+import { BackOffRequest, setBackOffModule } from '../../src/rest/BackOffRequest';
+import { ISuccessResponse } from '../../src/rest/EndpointCaller';
+
+export function BackOffRequestTest() {
+  describe('BackOffRequest', () => {
+    const oneTimeUnit = 100;
+    const mockSuccessResponse: ISuccessResponse<{}> = {
+      duration: 1,
+      data: {}
+    };
+
+    beforeAll(() => stubBackOffDependency());
```

Comment:
Shouldn't we make sure that we change the dependency back to the original module on `afterEach()`?